checkov/terraform/checks/resource/azure/SQLServerAuditPolicyRetentionPeriod.py

```diff
@@ -0,0 +1,24 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.common.util.type_forcers import force_int
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
+
+
+class SQLServerAuditPolicyRetentionPeriod(BaseResourceCheck):
+    def __init__(self):
+        name = "Specifies a retention period of less than 90 days."
+        id = "CKV_AZURE_46"
+        supported_resources = ['azurerm_mssql_database_extended_auditing_policy']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if 'retention_in_days' in conf: 
+            retention=force_int(conf['retention_in_days'][0])
+            if retention < 90:
```

Comment:
force_int might return None if the value is not a number or a stringified number. Need to handle the None case

Reply:
Good catch thanks!, added a catch for the none case.